nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/config/Configs.scala

```diff
@@ -365,6 +340,24 @@ object Configs {
     }
     parser.parse(args, Argument())
   }
+
+  def readConfig(config: Config, name: String): Map[String, String] = {
+    val map         = mutable.Map[String, String]()
+    val configObject = config.getObject(name)
+    for (key <- configObject.unwrapped().keySet().asScala) {
+      val refinedKey = s"$name.$key"
+      config.getAnyRef(refinedKey) match {
+        case stringValue: String => map += refinedKey -> stringValue
+        case _ =>
+          for (subKey <- config.getObject(refinedKey).unwrapped().keySet().asScala) {
+            val refinedSubKey        = s"$refinedKey.$subKey"
+            val sparkValue = config.getString(refinedSubKey)
```

Comment:
can we refactor the `sparkValue` to other variable name? such as `refinedSubValue`